src/styles/blocks/container.scss

```diff
@@ -0,0 +1,48 @@
+.container {
+  background-color: black;
```

Comment:
no good idea to add black here, you need to color blocks, not the background of the main container